### PR TITLE
Added information about image promotion with skopeo (#277)

### PR DESCRIPTION
**Pull request by srang**
#### What is this PR About?
Add brief introduction and example of image promotion using skopeo

#### How should we test or review this PR?
Just documentation, let me know if I got it wrong

#### Is there a relevant Trello card or Github issue open for this?
#277 

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

File: playbooks/continuous_delivery/image_promotion.adoc [hunk 1]
```diff
@@ -119,3 +119,19 @@ docker push <promoted_docker_registry>/<ose_project>/<ose_app>
 docker rmi <origin_docker_registry>/<ose_project>/<ose_app>
 docker rmi <promoted_docker_registry>/<ose_project>/<ose_app>
 ----
+
+
+=== Promotion Process with `skopeo`
+
+link:https://github.com/projectatomic/skopeo[Skopeo] is an opensource command line tool maintained by the Project Atomic group. It works with images in remote repositories without the need to pull the images to the host. With `skopeo` the promotion process is much more streamlined:
```
Comment (sabre1041) [inline playbooks/continuous_delivery/image_promotion.adoc]: Split out open source into 2 words. Add a link to the Project Atomic homepage.

I would also emphasize skopeo as a tool that does not require privileged access to manipulate container images which is a contrast to docker

Comment (srang) [reply playbooks/continuous_delivery/image_promotion.adoc]: When you say privileged access, do you mean that the tool doesn't require elevated access on the intermediate host or something else?

**Comment by srang** [comment]
@sabre1041 are there more changes you were looking for?

**Comment by sabre1041** [comment]
@srang LTGM

**Comment by mwitzenm** [comment]
merge away

**Pull request merged**